### PR TITLE
243 Document Endpoints to get MVTs for Capital Data

### LINKED ISSUE #243: Open API: Document endpoints to get MVTs for capital data
# Description
Document endpoints that return MVTs for city council districts, community districts, and captial projects

## Acceptance criteria
- [x] MVT endpoints are created under [`paths`](https://github.com/NYCPlanning/ae-zoning-api/blob/9561c9bd8b80a8ca49f52505899bfec4500904e9/openapi/openapi.yaml#L15)
  - They are placed alphabetically within the list of paths, as practicable
- [x] The endpoints have a get method
- [x] The endpoints respond with
   - 200, which returns an MVT
   - 400, which returns a bad request
   - 500, which returns an internal server error
- [x] `/community-districts/{z}/{x}/{y}.pbf`
   - summary: Mapbox Vector Tiles for community districts
   - operationId: findCommunityDistrictTiles
   - tags
      - MVT
      -  Community Districts
- [x] `/capital-projects/{z}/{x}/{y}.pbf`
   - summary: Mapbox Vector Tiles for capital projects
   - operationId: findCapitalProjectTiles
   - tags
      - MVT
      -  Capital Projects
- [x] `/city-council-districts/{z}/{x}/{y}.pbf`
   - summary: Mapbox Vector Tiles for city council districts
   - operationId: findCityCouncilDistrictTiles
   - tags
      - MVT
      -  City Council Districts

## Helpful code snippet
[/tax-lots/labels/{z}/{x}/{y}.pbf](https://github.com/NYCPlanning/ae-zoning-api/blob/f2508c4d1059b35b1e5e295c10566545094c7982/openapi/openapi.yaml#L88)

## Related tickets
Blocked by #225

### PR DESCRIPTION
**Pull request by horatiorosa**
#### Description
Endpoints for mapbox tiles for capital data

#### Tickets
 - Closes #243

### CODE REVIEW

**Review by TangoYankee — COMMENTED** [review]
The code changes meet spec. Waiting for its turn in the queue of endpoints (#278 ) before approval.

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -89,6 +89,60 @@ paths:
           $ref: "#/components/responses/NotFound"
         '500':
           $ref: "#/components/responses/InternalServerError" 
+  /capital-projects/{z}/{x}/{y}.pbf:
+    get:
+      summary: Mapbox Vector Tiles for capital projects
```
Comment (TangoYankee) [inline openapi/openapi.yaml:94]:
```suggestion
      summary: 🚧 Mapbox Vector Tiles for capital projects
```

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -89,6 +89,60 @@ paths:
...
+      responses:
+        '200':
+          $ref: '#/components/responses/MVT'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /city-council-districts/{z}/{x}/{y}.pbf:
+    get:
+      summary: Mapbox Vector Tiles for city council districts
```
Comment (TangoYankee) [inline openapi/openapi.yaml:112]:
```suggestion
      summary: 🚧 Mapbox Vector Tiles for city council districts
```

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -89,6 +89,60 @@ paths:
...
+      responses:
+        '200':
+          $ref: '#/components/responses/MVT'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /community-districts/{z}/{x}/{y}.pbf:
+    get:
+      summary: Mapbox Vector Tiles for community districts
```
Comment (TangoYankee) [inline openapi/openapi.yaml:130]:
```suggestion
      summary: 🚧 Mapbox Vector Tiles for community districts
```

**Review by TangoYankee — CHANGES_REQUESTED** [review]

**Comment by horatiorosa** [comment]
@TangoYankee  check the alphabetization order. Let me know if that is OK.

**Comment by TangoYankee** [comment]
> @TangoYankee check the alphabetization order. Let me know if that is OK.

Mmm, trying to alphabetize based on path parameters is weird. The way you have it seems fine.

**Review by TangoYankee — APPROVED** [review]

**Pull request merged**